packages/x-components/src/views/home/result.vue

```diff
@@ -35,29 +35,34 @@
 </template>
 
 <script lang="ts">
-  import Vue from 'vue';
-  import { Component, Prop } from 'vue-property-decorator';
+  import { defineComponent } from 'vue';
   import { Result } from '@empathyco/x-types';
   import BaseResultAddToCart from '../../components/result/base-result-add-to-cart.vue';
   import BaseResultImage from '../../components/result/base-result-image.vue';
   import BaseResultLink from '../../components/result/base-result-link.vue';
   import BaseResultRating from '../../components/result/base-result-rating.vue';
   import CrossFade from '../../components/animations/cross-fade.vue';
 
-  @Component({
+  export default defineComponent({
+    name: 'Result',
     components: {
       BaseResultAddToCart,
       BaseResultLink,
       BaseResultRating,
       BaseResultImage
+    },
+    props: {
+      result: {
+        type: Object as () => Result,
```

Comment:
This prop type definition is ok, but you can also define the type with 
`Object as PropType<Result>`